<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>889</width>
    <height>538</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>15</pointsize>
    <weight>50</weight>
    <bold>false</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>Кредитный калькулятор</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>871</width>
      <height>261</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe Print</family>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="3" column="7" colspan="2">
      <widget class="QLabel" name="label_6">
       <property name="font">
        <font>
         <family>Segoe Print</family>
         <pointsize>15</pointsize>
         <weight>75</weight>
         <italic>true</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="1" colspan="2">
      <widget class="QLabel" name="label_2">
       <property name="font">
        <font>
         <family>Segoe Print</family>
         <pointsize>15</pointsize>
         <weight>75</weight>
         <italic>true</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Срок кредита:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" colspan="2">
      <widget class="QLabel" name="label">
       <property name="maximumSize">
        <size>
         <width>192</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Segoe Print</family>
         <pointsize>15</pointsize>
         <weight>75</weight>
         <italic>true</italic>
         <bold>true</bold>
         <kerning>true</kerning>
        </font>
       </property>
       <property name="text">
        <string>Сумма кредита:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="6" colspan="3">
      <widget class="QLabel" name="label_23">
       <property name="font">
        <font>
         <family>Yu Gothic Light</family>
         <pointsize>15</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="5" column="1" colspan="2">
      <widget class="QLabel" name="label_9">
       <property name="font">
        <font>
         <family>Segoe Print</family>
         <pointsize>15</pointsize>
         <weight>75</weight>
         <italic>true</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Итоговая сумма:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="3" colspan="2">
      <widget class="QLabel" name="label_22">
       <property name="font">
        <font>
         <family>Yu Gothic Light</family>
         <pointsize>15</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="0" column="3" colspan="6">
      <widget class="QLineEdit" name="lineEdit">
       <property name="font">
        <font>
         <family>Yu Gothic Light</family>
         <pointsize>15</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: #DBD7D2;color:#000000</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="clearButtonEnabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="5">
      <widget class="QLabel" name="label_7">
       <property name="font">
        <font>
         <family>Segoe Print</family>
         <pointsize>15</pointsize>
         <weight>75</weight>
         <italic>true</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Месяцев</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3" colspan="2">
      <widget class="QSpinBox" name="spinBox">
       <property name="font">
        <font>
         <family>Yu Gothic Light</family>
         <pointsize>15</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: #DBD7D2;color:#000000</string>
       </property>
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::PlusMinus</enum>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>60</number>
       </property>
      </widget>
     </item>
     <item row="6" column="1" colspan="8">
      <widget class="QPushButton" name="pushButton">
       <property name="font">
        <font>
         <family>Segoe Print</family>
         <pointsize>15</pointsize>
         <weight>75</weight>
         <italic>true</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: #DBD7D2;color:#000000</string>
       </property>
       <property name="text">
        <string>Рассчитать</string>
       </property>
      </widget>
     </item>
     <item row="3" column="4">
      <widget class="QLabel" name="label_4">
       <property name="font">
        <font>
         <family>Segoe Print</family>
         <pointsize>15</pointsize>
         <weight>75</weight>
         <italic>true</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>% годовых</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1" colspan="2">
      <widget class="QLabel" name="label_3">
       <property name="font">
        <font>
         <family>Segoe Print</family>
         <pointsize>15</pointsize>
         <weight>75</weight>
         <italic>true</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Процентная ставка: </string>
       </property>
      </widget>
     </item>
     <item row="3" column="6">
      <widget class="QLabel" name="label_11">
       <property name="font">
        <font>
         <family>Segoe Print</family>
         <pointsize>15</pointsize>
         <weight>75</weight>
         <italic>true</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="5" column="5">
      <widget class="QLabel" name="label_12">
       <property name="font">
        <font>
         <family>Segoe Print</family>
         <pointsize>15</pointsize>
         <weight>75</weight>
         <italic>true</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Переплата:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="3" colspan="5">
      <widget class="QComboBox" name="comboBox">
       <property name="font">
        <font>
         <family>Yu Gothic UI Light</family>
         <pointsize>15</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: #DBD7D2;color:#000000</string>
       </property>
       <property name="maxVisibleItems">
        <number>2</number>
       </property>
       <item>
        <property name="text">
         <string>аннуитетный</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>дифференцированный</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="4" column="8">
      <widget class="QPushButton" name="pushButton_2">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="maximumSize">
        <size>
         <width>30</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Roboto Th</family>
         <pointsize>15</pointsize>
         <weight>75</weight>
         <italic>true</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: #DBD7D2;color:#000000</string>
       </property>
       <property name="text">
        <string>?</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1" colspan="2">
      <widget class="QLabel" name="label_8">
       <property name="font">
        <font>
         <family>Segoe Print</family>
         <pointsize>15</pointsize>
         <weight>75</weight>
         <italic>true</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Вид платежа:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="3">
      <widget class="QDoubleSpinBox" name="doubleSpinBox">
       <property name="font">
        <font>
         <family>Yu Gothic Light</family>
         <pointsize>15</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: #DBD7D2;color:#000000</string>
       </property>
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::PlusMinus</enum>
       </property>
       <property name="minimum">
        <double>1.000000000000000</double>
       </property>
       <property name="maximum">
        <double>35.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.500000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>280</y>
      <width>871</width>
      <height>221</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe Print</family>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item alignment="Qt::AlignHCenter">
      <widget class="QLabel" name="label_10">
       <property name="font">
        <font>
         <family>Segoe Print</family>
         <pointsize>15</pointsize>
         <weight>75</weight>
         <italic>true</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="text">
        <string>График платежей по кредиту</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTableWidget" name="tableWidget">
       <property name="font">
        <font>
         <family>Yu Gothic Light</family>
         <pointsize>15</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: white;</string>
       </property>
       <property name="horizontalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
       <property name="sizeAdjustPolicy">
        <enum>QAbstractScrollArea::AdjustToContents</enum>
       </property>
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <property name="tabKeyNavigation">
        <bool>false</bool>
       </property>
       <property name="showDropIndicator" stdset="0">
        <bool>false</bool>
       </property>
       <property name="dragDropOverwriteMode">
        <bool>false</bool>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::NoSelection</enum>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
       <property name="columnCount">
        <number>0</number>
       </property>
       <attribute name="horizontalHeaderCascadingSectionResizes">
        <bool>true</bool>
       </attribute>
       <attribute name="horizontalHeaderDefaultSectionSize">
        <number>150</number>
       </attribute>
       <attribute name="verticalHeaderVisible">
        <bool>true</bool>
       </attribute>
       <attribute name="verticalHeaderMinimumSectionSize">
        <number>20</number>
       </attribute>
       <attribute name="verticalHeaderDefaultSectionSize">
        <number>20</number>
       </attribute>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>889</width>
     <height>33</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
